tools/clang/test/HLSLFileCheck/hlsl/objects/Texture/cube_gather.hlsl

```diff
@@ -18,64 +18,65 @@ float4 main(float4 a : A) : SV_Target
   ///////////////////////////////////////////////
 
   // CHECK: call %dx.types.ResRet.f32 @dx.op.textureGather.f32(i32 73,
-  // CHECK: , float %{{[^,]+}}, float %{{[^,]+}}, float %{{[^,]+}}, float undef
-  // CHECK: , i32 undef, i32 undef
-  // CHECK: , i32 0)
+  // CHECK-SAME: , float %{{[^,]+}}, float %{{[^,]+}}, float %{{[^,]+}}, float undef
```

Comment:
Changed all these to CHECK-SAME because debugging the test when it failed was much harder without it.